Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -2151,6 +2153,25 @@ TextStream& operator<<(TextStream& ts, const WebProcessProxy& process)
     return ts;
 }
 
+void WebProcessProxy::restartPeriodicalResponsivenessTimer()
+{
+    m_periodicalResponsivenessTimer->restart();
+}
+
+void WebProcessProxy::startOrStopPeriodicalResponsivenessTimer()
+{
+    bool shouldBeStopped = true;
+    for (auto* page : pages()) {
```

Comment:
I'd recommend iterating over `for (auto& page : m_pageMap.values()) {`. It'd be a little more efficient as it would avoid creating a vector of pages.